packages/templates/plugin/typescript/src/index.ts

```diff
@@ -35,3 +40,12 @@ export class SamplePlugin extends Plugin {
     return this._config.defaultValue;
   }
 }
+
+export const samplePlugin: PluginFactory<SamplePluginConfig> = (
```

Comment:
Minot nit, but I think each plugin module should export its PluginFactory by the name `plugin`. In the future this will help us to dynamically import plugin module, and know that there should be an export named `plugin` that represents its factory.

We could do this as a default module export as well, but I prefer having the named export. Just personal opinion.

Reply:
When you import multiple, you'll just have to rename the import like so `import { plugin as ethereumPlugin } from "..."`

Reply:
Renaming every plugin import is quite bad dev experience. Maybe export both named as samplePlugin and default export for dynamic importing?